### DISCUSSION TITLE
罪恶都市没有声音

### DISCUSSION THREAD

**Discussion by EnderCHX** [Gamuxorg/bbs#214] [游戏或网站运行反馈]
游戏名称：罪恶都市

操作系统：manjaro

bug描述：没有声音

终端运行输出 ：
[DBG]: cdvd_stream: read info 0x55ee986fc020
[DBG]: Using one streaming thread for all channels
[DBG]: size of matrix 80
[DBG]: size of placeable 80
[DBG]: size of entity 120
[DBG]: size of building 120
d
[DBG]: size of building 120
[DBG]: size of dummy 128
[DBG-2]: [/home/runner/work/re3/re3/src/skel/glfw/glfw.cpp.psInitialize:396]: gGameState = GS_START_UP
casepath couldn't find dir/file "gta_vc.set", full path was gta_vc.set
[DBG-2]: Default skin set as no other skins are available OR saved skin not found!
[DBG]: Physical memory size 2988191744
[DBG]: Available physical memory 330416128
OpenGL version: 4.6 (Core Profile) Mesa 21.1.6
casepath couldn't find dir/file "gta_vc.set", full path was gta_vc.set
[DBG-2]: [/home/runner/work/re3/re3/src/skel/glfw/glfw.cpp.main:1764]: gGameState = GS_INIT_ONCE
[DBG]: Initialising CTimer...
[DBG-2]: Performance counter not available, using millesecond timer

[DBG]: CTimer ready
[DBG]: Initialising CPedStats...
[DBG]: Loading pedstats data...
[DBG]: CPedStats ready
[DBG]: Intialising CTimeCycle...
[DBG]: CTimeCycle ready
ALSA lib pcm_dmix.c:1035:(snd_pcm_dmix_open) unable to open slave
[DBG]: Loadind audio cache (If game crashes around here, then your cache is corrupted, remove audio/sound.cache)
[DBG-2]: [/home/runner/work/re3/re3/src/skel/glfw/glfw.cpp.main:1857]: gGameState = GS_INIT_FRONTEND;
[DBG-2]: [/home/runner/work/re3/re3/src/skel/glfw/glfw.cpp.main:1879]: gGameState = GS_FRONTEND;
casepath couldn't find dir/file "GTAVCsf2.b", full path was userfiles\GTAVCsf2.b
casepath couldn't find dir/file "GTAVCsf3.b", full path was userfiles\GTAVCsf3.b
casepath couldn't find dir/file "GTAVCsf4.b", full path was userfiles\GTAVCsf4.b
casepath couldn't find dir/file "GTAVCsf5.b", full path was userfiles\GTAVCsf5.b
casepath couldn't find dir/file "GTAVCsf6.b", full path was userfiles\GTAVCsf6.b
casepath couldn't find dir/file "GTAVCsf7.b", full path was userfiles\GTAVCsf7.b
casepath couldn't find dir/file "GTAVCsf8.b", full path was userfiles\GTAVCsf8.b
LOAD frontend1
LOAD frontend2

**Comment by shouhuanxiaoji** [comment]
哪个架构

**Comment by EnderCHX** [comment]
x86_64

**Comment by Leilei332** [comment]
添加Steam运行库试试

**Comment by shouhuanxiaoji** [comment]
近期解决
